### PR TITLE
[AB#208629]-add new column in payments table

### PR DESCRIPTION
**Pull request by mmaciekk**
[AB#208629](https://unicef.visualstudio.com/ICTD-HCT-MIS/_boards/board/t/HOPE%20Team/Stories/?workitem=208629)

### CODE REVIEW

File: src/frontend/src/containers/tables/paymentmodule/PaymentsTable/PaymentsTableRow.tsx [hunk 1]
```diff
@@ -99,6 +93,17 @@ export function PaymentsTableRow({
     );
   };
 
+  const renderPaymentStatusBox = (): React.ReactElement => {
```
Comment (MarekBiczysko) [inline src/frontend/src/containers/tables/paymentmodule/PaymentsTable/PaymentsTableRow.tsx:96]: Shouldn't we display all possible statuses?

Comment (mmaciekk) [reply src/frontend/src/containers/tables/paymentmodule/PaymentsTable/PaymentsTableRow.tsx:96]: yes WE should.

Comment (mmaciekk) [reply src/frontend/src/containers/tables/paymentmodule/PaymentsTable/PaymentsTableRow.tsx:96]: fixed:
image

**Review by pavlo-mk — APPROVED** [review]

**Review by pkujawa — APPROVED** [review]

**Comment by domdinicola** [comment]
@mmaciekk pls resolve conflicts

**Pull request merged**